README.md

```diff
@@ -71,6 +79,25 @@ npm run serve
     npm run run:${PLATFORM}
     ```
 
-# Tech notes
-
--   Use interfaces to [define prop types](https://github.com/facebook/create-react-app/issues/8021) in typescript
+# Guidelines
+
+- JS & Typescript: 
+    - use destructuring
+    - use TS types & interfaces 
+    - small to medium (1 -> ~300 lines) components
+    - reusable components should be within `src/components`
+    - reducers/actions/selectors should live alongside and be scoped to the domain (possible domains: room/call, dashboard/stats, group)
+    - actions,reducers, props should be typed
+- Libraries:
+    - keep the amount of libraries to a minimum to have the lowest possible bundle size
+    - form: Formik, Yup
+    - UI: Material UI
+    - state: Redux & redux-thunk for async actions
+    - routing: Ionic Router
+- Code organization
+    - draft Pull Request for early on review if needed
```

Comment:
`draft` avoids to send notifications to the reviewers actually (unless already subscribed to the PR). So I think:
- `draft` should mean *no review needed for the moment*
- reviews should be made only when a PR is ready (generally)
- to reason about architecture, we may better discuss that into the related ticket, before implementation
- in case an early review is still needed, we may specify that in a PR comment ? Maybe tagging the reviewers ?